app/controllers/api/functions.php

```diff
@@ -389,37 +601,147 @@
     ->label('sdk.response.model', Response::MODEL_FUNCTION)
     ->param('functionId', '', new UID(), 'Function ID.')
     ->param('name', '', new Text(128), 'Function name. Max length: 128 chars.')
+    ->param('runtime', '', new WhiteList(array_keys(Config::getParam('runtimes')), true), 'Execution runtime.')
```

Comment:
Is this supposed to be required?

Reply:
It's PUT, not PATCH. It must include all info of resource. Since runtime is required on create, it should be required on update as well (as far as I understand REST)

Reply:
oh wow...this is going to be a problem for the request filter 👀

Reply:
Closing this, we'll continue the internal discussion.